tests/e2e/Services/Databases/DatabasesCustomServerTest.php

```diff
@@ -1557,6 +1568,17 @@ public function testAttributeUpdateEmail(array $data)
         $this->assertFalse($new['body']['required']);
         $this->assertEquals('torsten@appwrite.io', $new['body']['default']);
 
+        $new = $this->client->call(Client::METHOD_GET, '/databases/' . $databaseId . '/collections/' . $collectionId, array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]));
+
+        $attribute = array_values(array_filter($new['body']['attributes'], fn (array $a) => $a['key'] === $key))[0] ?? null;
```

Comment:
I think array_values issue was fixed by @abnegate or can be fixed before saving the metadata attributes?